### ISSUE TITLE
Epistemic uncertainty for GraphConvModel

### ISSUE THREAD

**Issue by alexkoetter** [deepchem/deepchem#3417]
## To Reproduce

You have to modify the predict_uncertainty method of KerasModel to separately return epistemic and aleatoric contribution (very simple, just return the summands separately). Then train a GraphConvModel, e.g., on the delaney data for some epochs:

`model = GraphConvModel(n_tasks=1, mode="regression", batch_size=50, uncertainty=True, dropout=0.5)`
`model.fit(train_dataset, nb_epoch=10)`

The estimated epistemic uncertainties are basically zero for all test molecules (within -1e-6 and 1e-6).

## Expected behavior

For high dropout ratios I would expect non-vanishing epistemic contribution to the uncertainty.

## Environment

* OS: MATE 1.16.2
* Python version: 3.9.13
* DeepChem version: 2.6.1
* TensorFlow version (optional): 2.13.0-rc0
